src/translation-assistant.js

```diff
@@ -375,18 +377,43 @@ function translateMath(math, lang) {
 
     const mathTranslations = [
          // division sign as a colon
-         {langs: ['cs', 'de'],
-            regex: /\\div/g, replace: '\\mathbin{:}'},
+         {langs: ['cs', 'de', 'bg', 'hu'],
+            regex: /\\div/g, replace: '\\mathbin\{:\}'},
+
          // latin trig functions
          {langs: ['es', 'it', 'pt', 'pt-pt'],
             regex: /\\sin/g, replace: '\\operatorname\{sen\}'},
+
+         // multiplication sign as a centered dot
+         {langs: ['cs', 'pl', 'de', 'nb', 'sr', 'ro', 'hu', 'sv'],
+            regex: /\\times/g, replace: '\\cdot'},
+
+         // multiplication sign as a simple dot, a Bulgarian specialty
+         // TODO(danielhollas): not yet allowed by the linter
+         // TODO(danielhollas): add a test for this case
+         //{langs: ['bg'],
+         //   regex: /\\times/g, replace: '\\mathbin\{.\}'},
+
+         // Thousand separator as a thin space
+         {langs: ['cs', 'fr', 'de', 'pt-pt', 'nb', 'bg', 'pl', 'ro', 'nl',
+                  'az', 'se', 'it', 'hu'],
+            regex: /([0-9])\{,\}([0-9])(?=[0-9]{2})/g, replace: '$1\\,$2'},
+
+         // No thousand separator
+         {langs: ['ko'],
+            regex: /([0-9])\{,\}([0-9])(?=[0-9]{2})/g, replace: '$1$2'},
+
+         // Thousand separator as a dot
+         // NOTE(danielhollas): Extra braces around the dot are needed
```

Comment:
Is one of the tests you added fail if the order of regexes is changed to break this? If not, perhaps it's possible to add a test case to catch if someone mistakingly does change the order here?

nit: Maybe `// IMPORTANT NOTE(danielhollas): Extra...` just to make it more visible?

Reply:
I believe that because of the extra braces, the order of regexes should not matter in this case, since English decimal dot is not enclosed by them.